ama_layout.gemspec

```diff
@@ -4,31 +4,33 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'ama_layout/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "ama_layout"
+  spec.name          = 'ama_layout'
```

Comment:
Yea... I see files with both styles and something in the back of my head makes me unify the file!